### ISSUE TITLE
ML Gestures: Create ML activity classification model

### ISSUE THREAD

**Issue by meninoebom** [meninoebom/gesture-input-ios#9]
## Description
Train and deploy a Create ML activity classification model for more sophisticated gesture recognition.

## Tasks
- [ ] Design gesture vocabulary (5-10 distinct gestures)
- [ ] Build data collection mode in app
- [ ] Collect training data (50+ samples per gesture)
- [ ] Create ML project in Xcode
- [ ] Train activity classification model
- [ ] Export Core ML model (.mlmodel)
- [ ] Integrate model into GestureDetector
- [ ] Implement windowed inference (prediction every 0.5s)
- [ ] Add confidence threshold filtering
- [ ] Test and iterate on accuracy

## Gesture Vocabulary (Suggested)
1. `idle` - no gesture
2. `shake` - shaking motion
3. `circle_cw` - clockwise circle
4. `circle_ccw` - counter-clockwise circle
5. `swipe_left` - left swipe
6. `swipe_right` - right swipe
7. `swipe_up` - upward swipe
8. `swipe_down` - downward swipe
9. `tap` - quick tap motion
10. `wave` - waving motion

## Technical Notes
```swift
// Load model
let config = MLModelConfiguration()
let model = try GestureClassifier(configuration: config)

// Inference
func classifyGesture(samples: [MotionData]) -> (String, Double)? {
    let multiArray = createMultiArray(from: samples)
    let prediction = try model.prediction(input: multiArray)
    return (prediction.label, prediction.labelProbabilities[prediction.label] ?? 0)
}
```

## Acceptance Criteria
- Model achieves >90% accuracy on test set
- Inference runs in <50ms on device
- False positive rate <5%
- Works offline (on-device inference)

## References
- [WWDC19: Building Activity Classification Models](https://developer.apple.com/videos/play/wwdc2019/426/)
- [Create ML Documentation](https://developer.apple.com/documentation/createml)

## Dependencies
- #8 Basic Gesture Detection (for data collection UI)
